tests/ui/imports/reexports.stderr

```diff
@@ -62,7 +62,7 @@ warning: glob import doesn't reexport anything with visibility `pub` because no
 LL |         pub use super::*;
    |                 ^^^^^^^^
    |
-note: the most public imported item is `pub(a)`
+note: the most public imported item is `pub(in crate::a)`
```

Comment:
My change seems to affect here but I'm not sure this is correct. Should I keep as it was before (`pub(a)`)?

Reply:
It seems correct to me